src/components/EnvironmentOptions.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { TerminalIcon, VSCodeIcon } from "./Icons";
+
+const options = [
+  {
+    title: "Runme VS Code extension",
+    href: "/getting-started/vs-code",
+    icon: VSCodeIcon,
+  },
+  {
+    title: "Runme CLI",
+    href: "/getting-started/cli",
+    icon: TerminalIcon,
+  },
+];
+
+export default function EnvironmentOptions() {
+  return (
+    <div className="flex gap-1.5 justify-center items-center flex-row p-2 m-2">
+      {options.map(({ href, title, icon }) => {
+        return (
+          <a
+            className="text-center shadow-md p-2 min-w-fit rounded"
```

Comment:
Some more spacing between the boxes and maybe have the same width?
```suggestion
            className="text-center shadow-md p-2 m-4 min-w-[200px] rounded"
```